qe_metrics/utils/issue_utils.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+from datetime import date, datetime
+
+from jira import Issue
+
+from simple_logger.logger import get_logger
+from typing import TYPE_CHECKING, List
+
+if TYPE_CHECKING:
```

Comment:
What this mean?

Reply:
@myakove `from qe_metrics.libs.database import Database` will cause a circular import but `Database` is used in the type annotations. This is essentially saying, only import if pre-commit hooks are running to check type annotations.